src/app/modules/main/networks/item.nim

```diff
@@ -71,4 +75,7 @@ proc getNativeCurrencySymbol*(self: Item): string =
   return self.nativeCurrencySymbol
 
 proc getIsTest*(self: Item): bool =
-  return self.isTest
\ No newline at end of file
+  return self.isTest
+
+proc getIsEnabled*(self: Item): bool =
```

Comment:
Just an FYI, you can name functions the same as the property, and since you can call functions in nim without parentheses, it looks like your accessing the property directly, but you're using the getter.

eg:
```nim
proc enabled*(self: Item): bool =
  result = self.enabled
  
# Also has a funny syntax for setters if needed
proc `enabled=`*(self: Item, value: bool) =
  self.enabled = value
```
Then you can do `let enabled = item.enabled`

Anyway, I just find that cuter, but no need to change anything.